### ISSUE TITLE
`into_json` uses field names set by `column_name`

### ISSUE THREAD

**Issue by e-rhodes** [SeaQL/sea-orm#906]
## Description
When getting data from `into_json`, the resulting `Value` uses the field names set by the `#[sea_orm(column_name = "")]` macro instead of the names of the fields in the struct.

## Steps to Reproduce

1. Prepare a query.
2. Call `into_json()`.
3. Retrieve the models from the database.

### Expected Behavior

I would expect 
```rust
    Entity::find().into_json().one(&conn).await.unwrap().unwrap();
```
to be equal to
```rust
    serde_json::to_value(Entity::find().one(&conn).await.unwrap()).unwrap();
```

If this is intended behavior, I'd at least like to be able to configure it so that they do produce the same output.

### Actual Behavior

The results from `into_json` use field names from `column_name`, while the results from `into_model` use the field names of the struct.

### Reproduces How Often

Every time

## Versions

```
─ sea-orm v0.9.1
├── sea-orm-macros v0.9.1 (proc-macro)
├── sea-query v0.26.2
│   ├── sea-query-derive v0.2.0 (proc-macro)
│   ├── sea-query-driver v0.2.0 (proc-macro)
├── sea-strum v0.23.0
│   └── sea-strum_macros v0.23.0 (proc-macro)
```
WSL Ubuntu 22.04 on Windows 11

## Additional Information

<https://github.com/e-rhodes/sea-orm-serialize-mre>

**Comment by billy1624** [comment]
Hey @e-rhodes, these two isn't equivalent:

1. Query result is directly mapped into a JSON. Without first deserializing into `Model` then serialized back into JSON. This is a performance optimization provided by SeaORM.
    ```rust
    Entity::find().into_json().one(&conn).await.unwrap().unwrap();
    ```
2. Query result is deserialized into `Model` and then serialized into JSON.
    ```rust
    serde_json::to_value(Entity::find().one(&conn).await.unwrap()).unwrap();
    ```

Hence, if you want the resulting JSON key same as the name of model field. Please go with code snippet 2.

**Comment by e-rhodes** [comment]
Makes sense. Unfortunately, this isn't an option since I'm dealing with custom selects and the structure isn't predictable—so users will specify a field by one name and then see it in the result by another name. Hope I can find a workaround.

**Comment by billy1624** [comment]
Okay, I guess you can `.map(|json| ...)` then transform the key inside the JSON object

**Comment by tyt2y3** [comment]
The idea is that there can be some other operations following `find()`, may be "group by" or "column as", so the fields are not predictable and they will not be the same as the model.

But - if you only want to convert the keys into camelCase, then you can write a small json transformer anyway.
